### PR TITLE
Scan: patch to support snake case, uptime

### LINKED ISSUE #1554: Patch camel case to snake case variable in oracle
Patch variable in data source, oracle script, request.

### LINKED ISSUE #1555: Patch variable name(validator things)
We doesn't use `reportedValidators`,  `minimumValidators` , `requestValidators` anymore.
Let's change to the new one. Ask @taobun for a new name.

Change both UI and variable's name.

### PR DESCRIPTION
**Pull request by evilpeach**
Fixes #1554 Fixes #1555 

Note: Review this before #1542 #1587 

- Patch to support snake case variable
- Patch validator's validator name
- Support uptime on Validator Index and home page

### CODE REVIEW

File: scan/src/subscriptions/DataSourceSub.re [hunk 1]
```diff
@@ -30,8 +28,7 @@ module SingleConfig = [%graphql
     data_sources_by_pk(id: $id) @bsRecord {
       id @bsDecoder(fn: "ID.DataSource.fromJson")
       owner @bsDecoder(fn: "Address.fromBech32")
-      name
-      fee @bsDecoder(fn: "GraphQLParser.coins")
+    name
```
Comment (taobun) [inline scan/src/subscriptions/DataSourceSub.re]: indent?

File: scan/src/subscriptions/TxSub.re [hunk 2]
```diff
@@ -198,20 +198,20 @@ module Msg = {
       oracleScriptID: ID.OracleScript.t,
```
Comment (taobun) [inline scan/src/subscriptions/TxSub.re:198]: EditDataSource seem not to be fixed

File: scan/src/wallet/TxCreator.re [hunk 3]
```diff
@@ -56,7 +56,7 @@ type msg_payload_t = {
 };
 
 type account_result_t = {
-  accountNumber: int,
+  account_number: int,
```
Comment (taobun) [inline scan/src/wallet/TxCreator.re]: Why does this field need to change?

**Review by taobun — CHANGES_REQUESTED** [review]

File: scan/src/subscriptions/TxSub.re [hunk 4]
```diff
@@ -184,7 +184,7 @@ module Msg = {
 
     let decode = json =>
       JsonUtils.Decode.{
-        id: json |> field("oracleScriptID", ID.OracleScript.fromJson),
+        id: json |> field("oracle_script_id", ID.OracleScript.fromJson),
```
Comment (taobun) [inline scan/src/subscriptions/TxSub.re:187]: line 148 too

**Review by taobun — CHANGES_REQUESTED** [review]

**Review by taobun — APPROVED** [review]

**Review by tansawit — APPROVED** [review]

**Pull request merged**